.circleci/config.yml

```diff
@@ -111,14 +116,34 @@ jobs:
           name: "Publish Azure polkadot provider release on GitHub"
           command: |
             GO111MODULE=off go get github.com/tcnksm/ghr
-            make build-azure-provider
             VERSION=$(git describe --tags --abbrev=8 --exact-match 2>/dev/null || echo "${AZURE_POLCADOT_PROVIDER_VERSION}")
-            VERSION=${VERSION#*-}
-            echo $VERSION
+            [[ "${VERSION}" =~ "^azure" ]] || VERSION="${AZURE_POLCADOT_PROVIDER_VERSION}"
+            VERSION=${VERSION#*-} make build-azure-provider
             mv -v ./terraform-provider-polkadot ./terraform-provider-polkadot-azure
             ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -delete ${VERSION} ./terraform-provider-polkadot-azure
       - slack/status:
           fail_only: true
+    environment:
+      - AZURE_POLCADOT_PROVIDER_VERSION: 0.1.0
+
+  build_gpg_polkadot_provider:
+    docker:
+      - image: circleci/golang:1.15
+    steps:
+      - checkout
+      - run:
+          name: "Publish GCP polkadot provider release on GitHub"
+          command: |
+            GO111MODULE=off go get github.com/tcnksm/ghr
+            VERSION=$(git describe --tags --abbrev=8 --exact-match 2>/dev/null || echo "${GCP_POLCADOT_PROVIDER_VERSION}")
+            [[ "${VERSION}" =~ "^gcp" ]] || VERSION="${GCP_POLCADOT_PROVIDER_VERSION}"
+            VERSION=${VERSION#*-} make build-gcp-provider
+            mv -v ./terraform-provider-polkadot ./terraform-provider-polkadot-gcp
+            ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -delete ${VERSION} ./terraform-provider-polkadot-gcp
```

Comment:
Providers seems to be a standalone units. Can we store them in the separate repo?

Reply:
As discussed, removed into a separate issue